canvas_modules/harness/test_resources/parameterDefs/slider_paramDef.json

```diff
@@ -0,0 +1,209 @@
+{
+  "titleDefinition": {
+    "title": "Slider with Min Max"
+  },
+  "current_parameters": {
+    "slider": {
+      "value": 50,
+      "min": 10,
+      "max": 100,
+      "minLabel": "",
+      "maxLabel": "",
+      "step": 10
+    },
+    "slider_error":{
+      "value": 50,
+      "min": 10,
+      "max": 100,
+      "minLabel": "",
+      "maxLabel": "",
+      "step": 10
+    },
+    "hide": true,
+    "slider-disable": false
+  },
+  "parameters": [
+    {
+      "id": "slider-disable",
+      "type": "boolean"
+	  },
+    {
+      "id": "slider",
+      "type": "integer"
+	  },
+    {
+      "id": "slider_error",
+      "type": "object",
+      "required": true,
+      "default": {
```

Comment:
We can't have a value like this.  Should be `default: 50` or something like that.